src/main/java/com/lunark/lunark/properties/model/Property.java

```diff
@@ -17,6 +17,7 @@
 import java.util.Collection;
 import java.util.Set;
 
+import lombok.Getter;
```

Comment:
If you apply my suggestion, you'll need additional imports.
```suggestion
import lombok.Getter;
import com.lunark.lunark.auth.model.Account;
import com.lunark.lunark.auth.model.AccountRole;
```